x-pack/plugins/observability/public/components/shared/alert_search_bar/alert_search_bar.test.tsx

```diff
@@ -5,57 +5,51 @@
  * 2.0.
  */
 
-import { triggersActionsUiMock } from '@kbn/triggers-actions-ui-plugin/public/mocks';
 import React from 'react';
 import { act, waitFor } from '@testing-library/react';
-import { AlertSearchBarProps } from './types';
+import { timefilterServiceMock } from '@kbn/data-plugin/public/query/timefilter/timefilter_service.mock';
+import { useServices } from './services';
+import { ObservabilityAlertSearchBarProps } from './types';
 import { ObservabilityAlertSearchBar } from './alert_search_bar';
 import { observabilityAlertFeatureIds } from '../../../config';
-import { useKibana } from '../../../utils/kibana_react';
-import { kibanaStartMock } from '../../../utils/kibana_react.mock';
 import { render } from '../../../utils/test_helper';
 
-const useKibanaMock = useKibana as jest.Mock;
+const useServicesMock = useServices as jest.Mock;
 const getAlertsSearchBarMock = jest.fn();
 const ALERT_SEARCH_BAR_DATA_TEST_SUBJ = 'alerts-search-bar';
-const ACTIVE_BUTTON_DATA_TEST_SUBJ = 'alert-status-filter-active-button';
-
-jest.mock('../../../utils/kibana_react');
-
-const mockKibana = () => {
-  useKibanaMock.mockReturnValue({
-    services: {
-      ...kibanaStartMock.startContract().services,
-      triggersActionsUi: {
-        ...triggersActionsUiMock.createStart(),
-        getAlertsSearchBar: getAlertsSearchBarMock.mockReturnValue(
-          <div data-test-subj={ALERT_SEARCH_BAR_DATA_TEST_SUBJ} />
-        ),
-      },
-    },
+
+jest.mock('./services');
+
+const mockServices = () => {
+  useServicesMock.mockReturnValue({
+    timeFilterService: timefilterServiceMock,
+    AlertsSearchBar: getAlertsSearchBarMock.mockReturnValue(
```

Comment:
I added `getAlertsSearchBar` to [the TriggerActionsUI mock over here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/observability_public_plugins_start.mock.tsx#L32). So you might not need to reintroduce this here.

Reply:
But this one is mocking `useServices` not `useKibana`, and I also want to check the related props, so I defined the mock here.